dev/snippets/nav/bar_basic.cljs

```diff
@@ -1,11 +1,14 @@
 #_
 (:require [om-bootstrap.button :as b]
-          [om-bootstrap.nav :as n])
+          [om-bootstrap.nav :as n]
+          [om-tools.dom :as d :include-macros true])
 
 (n/navbar
- {}
+ {:brand (d/a {:href "#"}
+              "Navbar")
+  :toggle-nav-key 0}
  (n/nav
-  {:collapsible? true}
+  {:key 0}
```

Comment:
So if you want to be able to just add :collapsible? true to a Nav, we have to make changes in those two locations. The change in `render-navbar-child` would be to also check for `(:collapsible? bs)`, and the change in `render-header` would be to also check for `(:brand bs)`. That's probably the right thing to do, since the main `navbar*` component also checks for that when decided whether to render a header.
```
             (when (or (:brand bs)
                             (:toggle-button bs)
                            (:toggle-nav-key bs))
                (render-header owner bs))
```